Telemetry/TelemetryStorage.swift

```diff
@@ -91,7 +107,7 @@ public class TelemetryStorage {
             return
         }
 
-        var url = directory.appendingPathComponent("\(Date().timeIntervalSince1970).json")
+        var url = directory.appendingPathComponent("-t-\(Date().timeIntervalSince1970).json")
```

Comment:
nit: is it necessary to prefix this?

Reply:
I added a key to the value so that we can store other numbers as global property on the file if needed in future and remove any ambiguity in parsing, I just used a single letter key for conciseness.